xlwings/udfs.py

```diff
@@ -59,6 +59,17 @@ async def async_thread(base, my_has_dynamic_array, func, args, cache_key, expand
     except:  # noqa: E722
         exception(logger, "async_thread failed")
 
+async def async_thread_nocaller(func, args,):
+    backcompat_check_com_initialized()
```

Comment:
this doesn't exist now anymore

Reply:
Sorry, I was not familiar with the pull interface in VSCode yet. How about now?

Reply:
Looks good now, thanks! will give this a look